src/js/covidMap.js

```diff
@@ -437,6 +537,97 @@ fluid.defaults("fluid.covidMap.map", {
                     record: "fluid.covidMap.filterCheckboxInMap"
                 }
             }
+        },
+        desktopViewHandler: {
+            source: "{map}.model.isDesktopView",
+            type: "fluid.covidMap.desktopViewHandler"
+        },
+        mobileViewHandler: {
+            source: "{map}.model.isMobileView",
+            type: "fluid.covidMap.mobileViewHandler"
+        }
+    }
+});
+
+fluid.covidMap.detectWindowSize = function (that) {
+    const mediaQuery = window.matchMedia(that.options.mediaQueryBreakpoint);
+    that.applier.change("isDesktopView", mediaQuery.matches ? true : false);
+    mediaQuery.addListener(function (evt) {
+        that.applier.change("isDesktopView", evt.matches ? true : false);
+    });
+};
+
+// The desktop view handler
+fluid.defaults("fluid.covidMap.desktopViewHandler", {
+    gradeNames: "fluid.modelComponent",
+    listeners: {
+        "onCreate.setVisiblePanels": {
+            listener: "{map}.applier.change",
+            args: ["visiblePanels", ["citiesList", "filterPanel"]]
+        },
+        "onCreate.setExpandedPanels": {
+            listener: "{map}.applier.change",
+            args: ["expandedPanels", {
+                citiesList: true,
+                resultsPage: true,
+                filterPanel: true,
+                hospitalPanel: true
+            }]
+        }
+    },
+    modelRelay: {
+        citiesShowing: {
+            source: "resultsShowing",
```

Comment:
I think this rule will be ineffective since neither the source nor target are on this component